packages/smart-contracts/src/contracts/ERC20TransferableReceivable.sol

```diff
@@ -35,6 +29,7 @@ contract ERC20TransferableReceivable is ERC721, ERC721Enumerable, ERC721URIStora
 
   /**
    * @notice Mapping for looking up a receivable given a paymentReference and minter address
+   * @dev Prevents minting multiple tokens for a given paymentReference and minter address
```

Comment:
Ever since #1091, the "minter" can be different from the initial owner. Please update the comment to make this clear. Also, please rearrange to match packing order `abi.encodePacked(owner, paymentReference)`

```suggestion
   * @notice Mapping for looking up a receivable given an initial owner address and paymentReference
   * @dev Prevents minting multiple tokens for a given initial owner address and paymentReference
```

Reply:
Good catch!